cli/ballerina-cli/src/test/resources/testng.xml

```diff
@@ -21,8 +21,24 @@ under the License.
 
 <suite name="commands-test-suite">
     <test name="command-tests" preserve-order="true" parallel="false">
-        <packages>
-            <package name="io.ballerina.cli.cmd.*"/>
-        </packages>
+        <!--<packages>-->
+            <!--<package name="io.ballerina.cli.cmd.*"/>-->
+        <!--</packages>-->
+        <classes>
```

Comment:
Why did we have to change this?

Reply:
There were some specific tests that were failing so rather than running the entire thing locally, I split them and ran them individually for convenience.

I accidentally included that in this pr. However, it still runs all the tests regardless. Shall i rever this section to use `*` instead?

Reply:
Reverted change to use `*` instead